### PR TITLE
14 spike increase code examples on the website

### PR DESCRIPTION
**Pull request by GCHQ-Developer-112**
## Summary of the changes
Add code examples for the following components:
- Breadcrumbs
- Cards
- Page header
- Radio
- Search bar
- Stepper

## Related issue
#14

## Checklist
- [x] I have manually accessibility tested any changes, if relevant.

### CODE REVIEW

File: src/content/structured/components/breadcrumbs/code.mdx [hunk 1]
```diff
@@ -51,3 +51,206 @@ export const snippetsDefault = [
 </ComponentPreview>
 
 <ComponentDetails component="ic-breadcrumb-group" />
+
+## Variants
+
```
Comment (GCHQ-Developer-530) [inline src/content/structured/components/breadcrumbs/code.mdx:56]: Sorry I forgot to add it to the list, but could we add a code example for react router?

Comment (GCHQ-Developer-530) [inline src/content/structured/components/breadcrumbs/code.mdx:56]: Sorry I forgot to add it to the list, but could we add a code example for react router?

Comment (GCHQ-Developer-112) [reply src/content/structured/components/breadcrumbs/code.mdx:56]: Yeah sure. I'm gonna break this PR down into two to hopefully get the ones that have no issues through

**Review by GCHQ-Developer-465 — COMMENTED** [review]
Linting is failing, other than that all is well.

**Pull request closed without merging**